src/compiler/core.ts

```diff
@@ -1941,41 +1939,28 @@ namespace ts {
      */
     export function getSpellingSuggestion<T>(name: string, candidates: T[], getName: (candidate: T) => string | undefined): T | undefined {
         const maximumLengthDifference = Math.min(2, Math.floor(name.length * 0.34));
-        let bestDistance = Math.floor(name.length * 0.4) + 1; // If the best result isn't better than this, don't bother.
+        let bestDistance = name.length * 0.4 + 0.1; // If the best result is worse than this, don't bother.

```

Comment:
Removing `Math.floor` is a more accurate implementation of `Whose levenshtein distance is more than 0.4 of the length of the name`. No test is broken by this, so I guess it should be acceptable?

Reply:
Can you put this in a followup change just like the Math.min/max one? I want this PR to be the minimal change so that future-me has less to read 2 years from now, next time we change this.

Reply:
👌